securedrop_client/gui/widgets.py

```diff
@@ -2232,6 +2232,7 @@ def sender(self, sender: User) -> None:
 
         if self._sender:
             self.sender_icon.initials = self._sender.initials
+            self.sender_icon.setToolTip(self._sender.fullname)
```

Comment:
When you log in, the tooltip doesn't show up until you send a reply. Moving this line to the constructor under the rest of the `self.sender_icon` setup code will make it so that the tooltip exists whenever a `ReplyWidget` is instantiated.

Reply:
@creviera Please take a look at my messages on Gitter regarding a couple of questions around this :)

Reply:
As you pointed out in gitter, you're right that the you don't _have_ to send a reply to show the tooltip (you could wait until after a sync). Here's a clip of the client with your code changes:

![tooltip-does-not-show-up-until-sync](https://user-images.githubusercontent.com/4522213/140864536-bf3bc7a1-3144-4a27-9261-855b1842ee41.gif)

If you copy (not move, like I said earlier) `self.sender_icon.setToolTip(self._sender.fullname)` to the constructor, then you should see the sender's fullname right after log in:

```diff
diff --git a/securedrop_client/gui/widgets.py b/securedrop_client/gui/widgets.py
index 1023f66..cc73c57 100644
--- a/securedrop_client/gui/widgets.py
+++ b/securedrop_client/gui/widgets.py
@@ -2208,6 +2208,7 @@ class ReplyWidget(SpeechBubble):
         self.sender_icon.is_current_user = self._sender_is_current_user
         if self._sender:
             self.sender_icon.initials = self._sender.initials
+            self.sender_icon.setToolTip(self._sender.fullname)
         self._update_styles()
 
     @property
```